apm-agent-core/src/main/java/co/elastic/apm/impl/transaction/Db.java

```diff
@@ -82,7 +82,7 @@ public Db withStatement(@Nullable String statement) {
         this.statement = statement;
         return this;
     }
-
+ 
```

Comment:
remove space

Reply:
r u kidding? it's the change I just did in order to test the GitHub issue...

Reply:
I didn't see that when looking at the full change set.